code/datums/status_effects/drugs.dm

```diff
@@ -0,0 +1,101 @@
+	alert_type = /atom/movable/screen/alert/status_effect/high_blood_pressure
+
+/datum/status_effect/high_blood_pressure/on_apply()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/human_owner = owner
+		human_owner.physiology.bleed_mod *= 1.25
+
+/datum/status_effect/high_blood_pressure/on_remove()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/human_owner = owner
+		human_owner.physiology.bleed_mod *= 0.80
+
+/atom/movable/screen/alert/status_effect/high_blood_pressure
+	name = "High blood pressure"
+	desc = "Your blood pressure is real high right now ... You'd probably bleed like a stuck pig."
+	icon_state = "highbloodpressure"
+
+
+
+/datum/status_effect/seizure
+	id = "seizure"
+	tick_interval = -1
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/seizure
+
+/datum/status_effect/seizure/on_apply()
+	if(!iscarbon(owner))
+		return FALSE
+	var/amplitude = rand(1 SECONDS, 3 SECONDS)
+	duration = amplitude
+	owner.Jitter(50)
+	owner.Paralyze(duration)
+	owner.visible_message("<span class='warning'>[owner] drops to the ground as [owner.p_they()] start seizing up.</span>")
+	owner.visible_message("<span class='warning'>[pick("You can't collect your thoughts...", \
+	"You suddenly feel extremely dizzy...", \
+	"You cant think straight...", \
+	"You can't move your face properly anymore...")]</span>")
+	return TRUE
+
+/atom/movable/screen/alert/status_effect/seizure
+	name = "Seizure"
+	desc = "It would appear you are having a seisure right now"
+	icon_state = "paralysis"
+
+/datum/status_effect/stoned
+	id = "stoned"
+	duration = 10 SECONDS
+	alert_type = /atom/movable/screen/alert/status_effect/stoned
+	status_type = STATUS_EFFECT_REFRESH
+	var/original_eye_color
+
+/datum/status_effect/stoned/on_apply()
+	if(!ishuman(owner))
+		CRASH("[type] status effect added to non-human owner: [owner ? owner.type : "null owner"]")
+	var/mob/living/carbon/human/human_owner = owner
+	original_eye_color = human_owner.eye_color
+	human_owner.add_movespeed_modifier(MOVESPEED_ID_CANNABIS, TRUE, 100, override = TRUE, multiplicative_slowdown= -0.4, movetypes=(~FLYING)) //slows you down
+	human_owner.eye_color = BLOODCULT_EYE //makes cult eyes less obvious
+	human_owner.update_body() //updates eye color
+	ADD_TRAIT(human_owner, TRAIT_BLOODSHOT_EYES, type) //dilates blood vessels in eyes
+	ADD_TRAIT(human_owner, TRAIT_CLUMSY, type) //impairs motor coordination
+	SEND_SIGNAL(human_owner, COMSIG_ADD_MOOD_EVENT, "stoned", /datum/mood_event/stoned) //improves mood
+	human_owner.sound_environment_override = SOUND_ENVIRONMENT_DRUGGED //not realistic but very immersive
+	return TRUE
+
+/datum/status_effect/stoned/on_remove()
+	if(!ishuman(owner))
+		stack_trace("[type] status effect being removed from non-human owner: [owner ? owner.type : "null owner"]")
+	var/mob/living/carbon/human/human_owner = owner
+	human_owner.remove_movespeed_modifier(MOVESPEED_ID_CANNABIS, TRUE)
+	human_owner.eye_color = original_eye_color
```

Comment:
Won't this just remove the cult eye colours?

Reply:
The /stoned/ status was a contamination in the original monke pr, it wasnt meant to be in this one. For example, the "MOVESPEED_ID_CANNABIS" is here, which is probably from copy-pasting code in monke, not from porting exactly. 

(It is in entirely fixed in my cannabis port #8463 as well as cult and bloodshot eyes being separated, which was the issue you saw here)

Reply:
Thank you for bringing my attention to it. I didn't call /stoned/ anywhere, so it wouldn't have an effect, but it was an unintentional add-on nonetheless.